x-pack/legacy/plugins/rollup/public/search/rollup_search_strategy.ts

```diff
@@ -51,35 +58,34 @@ function shimHitsInFetchResponse(response) {
   });
 }
 
-export const rollupSearchStrategy = {
+export const getRollupSearchStrategy = (fetch: HttpSetup['fetch']): SearchStrategyProvider => ({
   id: 'rollup',
 
-  search: ({ searchRequests, Promise }) => {
```

Comment:
I removed `Promise` because it isn't passed here any more
https://github.com/elastic/kibana/blob/master/src/plugins/data/public/search/search_strategy/types.ts#L35